.gitignore

```diff
@@ -53,6 +53,6 @@ public/js/vendor*
 public/js/faux*
 public/js/frame-runner*
 public/css/main*
-
+webpack-bundle-stats.html
```

Comment:
Instead of putting this on the root, can we instead add a new folder, `.stats` maybe? That way we can add more information to that folder that we find useful.

Reply:
Currently, this file ends up in `/public/js`. There currently is no option for a different output path, it is set by `output.path` of `webpack.config`. It wouldn't take much to fork and add this feature though.

Reply:
So this just ignores the file in public/js?